src/stories/spinner.stories.js

```diff
@@ -29,10 +29,18 @@ export default {
 // Template
 const Template = (args) => {
   const spinner = document.createElement('cod-spinner');
-  spinner.setAttribute('data-type', args.type);
-  spinner.setAttribute('data-background-color', args.backgroundColor);
-  spinner.setAttribute('data-size', args.size);
-  spinner.setAttribute('data-display-type', args.displayType);
+  spinner.setAttribute('type', args.type);
+  spinner.setAttribute('background-color', args.backgroundColor);
+  if (args.size){
+    spinner.setAttribute('size', args.size);
+  } else {
+    delete spinner.size;
+  }
```

Comment:
This shouldn't be necessary because the `size` attribute will never have been set if we reach this else condition.

Reply:
Removing this else statement entirely should continue to work as expected.